samples/WorkerBindingSamples/WorkerBindingSamples.csproj

```diff
@@ -7,12 +7,14 @@
         <Nullable>enable</Nullable>
     </PropertyGroup>
     <ItemGroup>
+        <PackageReference Include="Azure.Messaging.ServiceBus" Version="7.14.0" />
```

Comment:
Shouldn't have to add this package reference, it should be coming through from the extension (Worker.Extensions.ServiceBus)